CHANGELOG.md

```diff
@@ -1,5 +1,133 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.5
+
+<!-- generated comparing v5.0.0-beta.4..next -->
+
+_Aug 24, 2021_
+
+Big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐛 Fixed lots of regressions and bugs to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)
+- 🗎 Improved the docs and the migration guide for upgrading to v5
+
+### `@material-ui/core@5.0.0-beta.5`
+
+#### Breaking changes
+
+- <!-- 36 --> [core] Update `.browserslistrc` file (#27788) @DanailH
```

Comment:
Wasn't sure how to document this breaking change. Should we list all browsers, or?

Reply:
I would include the list of supported platforms like https://next--material-ui.netlify.app/guides/migration-v4/#supported-browsers-and-node-versions does but compared to the previous 5.x beta.

Reply:
Updated.

Reply:
To fix the rendering of the code part? 

```suggestion
- [core] Update `.browserslistrc` file (#27788) @DanailH
```